controllers/base_controller.js

```diff
@@ -344,11 +343,20 @@ module.exports = function BaseControllerModule(pb) {
                 urlOpts.locale = locale;
             }
             var url = pb.UrlService.createSystemUrl(path, urlOpts);
+            url = self.processUrlWithSiteOrigin(url);
             val += '<link rel="' + relationship + '" hreflang="' + locale + '" href="' + url + '" />\n';
         });
         cb(null, new pb.TemplateValue(val, false));
     };
 
+    BaseController.prototype.processUrlWithSiteOrigin = function(url){
+        if (this.siteObj.origin && this.siteObj.prefix) {
+            var origin = this.siteObj.origin + "/" + this.siteObj.prefix;
+            url = url.replace(/[a-zA-Z]*.jobs.net/g, origin);
```

Comment:
I think it is not useful to restrict this assuming always a jobs.net subdomain. Please make sure the logic is able to replace any domain with the value that comes from the setting:
`https://www.whatever.domain/job/J123 -> https://someothervalue.com/mypath/job/J123`

Reply:
See for example https://regexr.com/3au3g